packages/ink-editor/src/index.tsx

```diff
@@ -45,14 +39,16 @@ export const InkEditor = (props: InkEditorProps): ReactElement => {
     lineNumbers: props.numbering ? 'on' : 'off',
   };
 
-  return (
-    <MonacoEditor
-      language="rust"
-      theme={props.darkmode ? 'vs-dark' : 'vs'}
-      value={code}
-      options={options}
-      onChange={handleChange}
-      editorDidMount={editorDidMount}
-    />
-  );
+  if (props.code)
```

Comment:
This check is needed, because when the editor is initialized with an empty string and then immediately overwritten by either the example code or the loaded contract from gist, all the code is highlighted for some reason.